File: ncarg2d/src/libncarg/tdpack/tdline.f
C
C $Id: tdline.f,v 1.4 2008-07-27 00:17:32 haley Exp $
C                                                                      
C                Copyright (C)  2000
C        University Corporation for Atmospheric Research
C                All Rights Reserved
C
C The use of this Software is governed by a License Agreement.
C
      SUBROUTINE TDLINE (UCP1,VCP1,WCP1,UCP2,VCP2,WCP2)
C
C This routine draws a line joining the projections of two points in
C 3-space.  The points are defined by their positions in 3-space.
C
C Project the two points.
C
        CALL TDPRPT (UCP1,VCP1,WCP1,XCP1,YCP1)
        CALL TDPRPT (UCP2,VCP2,WCP2,XCP2,YCP2)
C
C Draw the line joining the projected points.
C
        CALL LINE   (XCP1,YCP1,XCP2,YCP2)
C
C Done.
C
        RETURN
C
      END

File: ncarg2d/man/tdpack/tdline.m
.TH TDLINE 3NCARG "July 1997" UNIX "NCAR GRAPHICS"
.na
.nh
.SH NAME
TDLINE - Given the coordinates of two points in 3-space, draw the projection
of the line joining them.
.SH SYNOPSIS
CALL TDLINE (UCP1, VCP1, WCP1, UCP2, VCP2, WCP2)
.SH C-BINDING SYNOPSIS
#include <ncarg/ncargC.h>
.sp
void c_tdline(float ucp1, float vcp1, float wcp1, float ucp2, float vcp2,
float wcp2)
.SH DESCRIPTION
.sp
The arguments of TDLINE are as follows:
.IP "UCP1, VCP1, and WCP1" 8
(input expressions of type REAL) -
the coordinates of a point in 3-space.
.IP "UCP2, VCP2, and WCP2" 8
(input expressions of type REAL) -
the coordinates of another point in 3-space.
.SH C-BINDING DESCRIPTION 
The C-binding argument descriptions are the same as the FORTRAN 
argument descriptions.
.SH ACCESS
To use TDLINE or c_tdline, load the NCAR Graphics libraries ncarg, ncarg_gks,
and ncarg_c, preferably in that order. 
.SH SEE ALSO
Online:
tdclrs, tdctri, tddtri, tdgeti, tdgetr, tdgrds, tdgrid, tdgtrs, tdinit, tditri,
tdlbla, tdlbls, tdlnpa, tdmtri, tdotri, tdpack, tdpack_params, tdpara,
tdplch, tdprpa, tdprpi, tdprpt, tdseti, tdsetr, tdsort, tdstri, tdstrs
.SH COPYRIGHT
Copyright (C) 1987-2009
.br
University Corporation for Atmospheric Research
.br
The use of this Software is governed by a License Agreement.
